Makefile

```diff
@@ -28,11 +28,11 @@ MANPAGES_MD = $(wildcard docs/*.md)
 
 all: binary docs
 
-binary: skopeo
+binary: skopeo-build
 
 # Build a docker image (skopeobuild) that has everything we need to build.
 # Then do the build and the output (skopeo) should appear in current dir
-skopeo: cmd/skopeo
+skopeo-build: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
Ah, true.  In that case, please just fold this into the `binary` target and we don't need the `skopeo-build` one (two phony targets chaining) at all.